### DISCUSSION TITLE
Github developer pack

### DISCUSSION THREAD

**Discussion by arpankesh** [github/india#283] [Students]
What do we have to do, in order to get the Github student developer pack?
I mean, do we have to complete a specific course or any other thing?

**Comment by rathore111-web** [comment]
Firstly ADD (Microsoft Azure) skills to your linkedIn profile

**Reply by rathore111-web** [reply]
Then Sign into Microsoft Azure using your GitHub account . I was facing some issues Signing inn through my pc so i did it with my phone you can try other devices too

**Reply by 19C91A0503** [reply]
My name is Siddharth. I already linked my GitHub account to Microsoft azure long back in June when I participated in the Azure fundamentals one-day program held by Microsoft. Now I'm not getting azure credit

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
